ui/src/hooks/auth.tsx

```diff
@@ -1,65 +1,145 @@
 import * as React from "react";
-import { Redirect, Route, RouteProps } from "react-router-dom";
+import {
+  GoogleLoginResponse,
+  GoogleLoginResponseOffline,
+  useGoogleLogin,
+  useGoogleLogout,
+} from "react-google-login";
 
-import { User } from "../types";
+const CLIENT_ID =
```

Comment:
is this okay here or do we want to put this in the .env?

Reply:
Great point, I'll pull this in from .env